packages/jest-editor-support/src/Settings.js

```diff
@@ -60,6 +63,34 @@ export default class Settings extends EventEmitter {
     };
 
     this.configs = [this.settings];
+    this._jsonPattern = new RegExp(/^[\s]*\{/gm);
+  }
+
+  _parseConfig(text: string): void {
+    let settings = null;
+
+    try {
+      settings = JSON.parse(text);
```

Comment:
By the way: When starting up your vscode-jest branch, I always get a "failed to parse config" with `text` being just:
```
"
> cra-ts-test@0.1.0 test c:\Users\stroy\Projekte\_Playground\cra-ts-test
> react-scripts-ts test --env=jsdom "--showConfig"
"
```

However since `_parseConfig` also gets called with the real settings, the extensions work without flaw.